Dockerfile.ci-rp.dockerignore

```diff
@@ -10,4 +10,14 @@
 !pkg
 !swagger
 !test
-!vendor
\ No newline at end of file
+!vendor
+
```

Comment:
```suggestion
# Don't pay attention to locally built NPM artifacts - we'll bring our own
+pkg/portal/assets/v2/build
```

...I take it back. I'm so convinced by your improvements that I say we should safely _ignore_ the old way of doing things. This makes a statement: with the new process - we have zero dependency on the old NPM build, and it works the same! This is HUGE! :)

Reply:
Thanks for pointing, Update Dockerfile.ci-rp.dockerignore to ignore old NPM artifacts